### DISCUSSION TITLE
can't sign in to my microsoft account

### DISCUSSION THREAD

**Discussion by s1e1z** [ORelio/Minecraft-Console-Client#1945] [General]
Minecraft Login failed :Incorrect password, blacklisted IP or too many logins.
Not connected to any server. Use '/help' for help.

**Comment by MousePak** [comment]
high chance its a hidden method microsoft use to block VPN's 
some dutch servers are know are all blocked but MS over chat said they do not block anyone !!
Oddly the same scripts on a different IP worked 

Try running from home on an IP you know 100% connects using a vanilla minecraft client to check its not a code problem.

Others might have different advise but thats what I found last time I hit this problem.

**Reply by SlimeLord101** [reply]
I'm having the same issue and using my home IP, worked fine before I switched to migrated to Microsoft but it doesn't anymore. The client is not supporting it for me, I get the same error that s1e1z got, login failed.

**Reply by s1e1z** [reply]
i don't use vpn, so if i connect via mojang is works fine
